### PR TITLE
Unify argument names in view_markers and view_connectome

### LINKED ISSUE #2852: Discrepancy of argument names in view_markers and view_connectome
Following #2810 (discussion [here](https://github.com/nilearn/nilearn/pull/2810#issuecomment-838597255)), we noticed that arguments for `html_connectome.view_connectome` and `html_connectome.view_markers` have different names even though they reflect the same object: 

- in `view_markers`, arguments relative to the markers are prefixed with `marker_` (e.g `marker_coords`)
- in `view_connectome`, arguments relative to the markers are prefixed with `node_` (e.g `node_coords`)

My opinion is that we should converge to either one of them. What do you think?

Signatures of the two functions:

```python
def view_markers(marker_coords, marker_color=None, marker_size=5., marker_labels=None, title=None, title_fontsize=25)
```

```python
def view_connectome(adjacency_matrix, node_coords, edge_threshold=None, edge_cmap=cm.bwr, symmetric_cmap=True, linewidth=6., node_size=3., colorbar=True, colorbar_height=.5, colorbar_fontsize=25, title=None, title_fontsize=25)
```

### PR DESCRIPTION
**Pull request by RaphaelMeudec**
This PR fixes #2852 by removing any references to `markers` in the connectome plots. This implies changes in:

- html_connectome argument names and methods
- connectome template arguments

### CODE REVIEW

File: nilearn/plotting/html_connectome.py [hunk 1]
```diff
@@ -319,25 +319,25 @@ def view_connectome(adjacency_matrix, node_coords, edge_threshold=None,
     return _make_connectome_html(connectome_info)
 
 
-def view_markers(marker_coords, marker_color='auto', marker_size=5.,
-                 marker_labels=None, title=None, title_fontsize=25):
-    """Insert a 3d plot of markers in a brain into an HTML page.
+def view_nodes(node_coords, node_color='auto', node_size=5.,
```
Comment (NicolasGensollen) [inline nilearn/plotting/html_connectome.py:322]: We need to deprecate `view_markers` (see my comment in the issue).

**Review by NicolasGensollen — COMMENTED** [review]
Thanks @RaphaelMeudec ! :+1: 
I made a quick first pass and it looks good so far. My main comment is related to the deprecation of `view_markers` (to be removed in v0.10.0). We also need to update `doc/modules/references.rst` and add an entry in whatsnew.rst under "changes".
We also need to update examples using `view_markers` if any.

File: nilearn/plotting/html_connectome.py (hunk 1 above)
Comment (RaphaelMeudec) [reply nilearn/plotting/html_connectome.py:322]: Yes, I forgot about it sorry. Will take care of this now!

**Comment by jeromedockes** [comment]
sorry but I'm not sure I agree with renaming view_markers -> view_nodes.
it is an important backwards incompatible change for little benefit, and it results in inconsistent naming with add_markers and plot_markers.

**Comment by NicolasGensollen** [comment]
@jeromedockes do you mean that we should keep the fonction name as view_markers but rename it's parameters to use the "node convention"?

**Comment by jeromedockes** [comment]
> @jeromedockes do you mean that we should keep the fonction name as view_markers but rename it's parameters to use the "node convention"?

I think so, otherwise this consistency rabbit hole would lead us to also
renaming plot_markers and add_markers and it becomes too much IMO.

**Comment by RaphaelMeudec** [comment]
@jeromedockes @NicolasGensollen Closing this PR as discussed.

**Pull request closed without merging**